notes/stories/film/film-scifi-2000s.st.txt

```diff
@@ -1165,6 +1165,50 @@ sacrifice for a child [Billy gave his life to save Eric from the flying dinosaur
 husband and wife [Ellie Sattler and husband.]
 
 
+movie: Mimic 2 (2001)
+=====================
+
+:: Title
+Mimic 2
+
+:: Description
+Mimic 2 is a 2001 science fiction horror film, directed by Jean de Segonzac,
+with a script inspired by a short story of the same name by Donald A.
+Wollheim. The movie was a direct-to-DVD sequel to Mimic (1997), and was
+followed by Mimic 3: Sentinel (2003).
+
+Synopsis: It falls on an entomophilic school teacher to stop a murderous
+human mimicking cockroach-like bug.
+
+:: Date
+2001-07-17
+
+:: References
+https://en.wikipedia.org/wiki/Mimic_2
+
+:: Ratings
+3 <paul>
+
+:: Major Themes
+speculative evolution [A main novelty of the story is that a genetically engineered cockroach had evolved into a quasi-intelligent, human mimicking creature.]
+human pleasure [The schoolteacher Remi loved all things bug and would never stop talking about the them.]
+human vs. monster [It was up to the schoolteacher Remi and two of her students to exterminate a giant, human mimicking cockroach-like creature that was going around killing people.]
+monster [There was a giant cockroach-like creature going around ripping people's faces off.]
+law enforcement [One storyline follows Detective Klaski as he investigates a handful of what appear to be grisly murders.]
+murder [One storyline follows Detective Klaski as he investigates a handful of what appear to be grisly murders.]
+
+:: Minor Themes
+the teaching professions [Remi taught her elementary school class about insects.]
+what it is like in school [The elementary school student Sal read his report about bugs in front of his class.]
+coping with unwelcome romantic attention [Remi was creeped out when a flower-bearing and highly awkward Jason showed up unannounced at her apartment door. Remi brushed off her former student Nicky when he asked her out on a date.]
+obsession [Remi's love life suffered as a direct result of her extreme interest in all things bug. Her superior Morrie Deaver went so far as to claim that she was "pathologically unable to sustain a proximate intrapersonal relationship" owing to her fascination with insects.]
+endangered specie [Lincoln Trahn had sold endangered species on the black market. This somehow resulted in 200 species of arthropods going extinct.]
```

Comment:
Typo: https://www.themeontology.org/theme.php?name=endangered%20species

Reply:
Whoops. Fixed. Will squash and merge.